maps/~mapsystem/maps_unit_testing.dm

```diff
@@ -3,6 +3,8 @@
 	var/const/NO_VENT = 2
 	var/const/NO_SCRUBBER = 4
 
+	var/shuttle_atmos_expectation = TRUE
```

Comment:
Please add a comment about what this does; I guess it's the supply shuttle's on-station area, and true means do check?

Reply:
Comment added, also swapped it to use the UT define.